frontend/src/components/Search.tsx

```diff
@@ -85,6 +85,10 @@ const Search: FunctionComponent = () => {
       size="sm"
       data={results}
       value={query}
+      limit={30}
+      // TODO: uncomment following line after upgrade to Mantine 7.x or higher
+      // scrollAreaProps = {{type: auto}}
+      maxDropdownHeight={400}
```

Comment:
@JaiZed 

I dare to say you don't need this after Mantine is updated to 7. I will leave you to check again without this once the mantine 7 is merged.